testing-guidance/workflow-tests/scripting/lua-scripting-tests.md

```diff
@@ -0,0 +1,329 @@
+# Lua Scripting Workflow Tests
+
+Testing in this area should focus on the functionality of Lua Editor.
+
+## Common Issues to Watch For
+
+Test guidance will sometimes note specific issues to watch for. The common issues below should be watched for through all testing, even if unrelated to the current workflow being tested.
+1. Asset processor errors when loading/saving .lua files
+2. Warnings or assets that appear in the Log
+3. Unresponsive UI elements 
+
+### Platforms:
+
+- Windows
+- Linux
+
+### Docs:
+
+<ul>
+	<li>[O3DE Documentation: Writing Lua Scripts in Open 3D Engine](https://www.o3de.org/docs/user-guide/scripting/lua/)</li>
+	<li>[O3DE Documentation: Introduction to Open 3D Engine’s Lua Editor](https://www.o3de.org/docs/user-guide/scripting/lua/lua-editor/)</li>
+	<li>[O3DE Documentation: Basic Structure of a Lua Script](https://www.o3de.org/docs/user-guide/scripting/lua/basic-lua-script/)</li>
+	<li>[O3DE Documentation: Debugging Lua Scripts](https://www.o3de.org/docs/user-guide/scripting/lua/debugging-scripts/)</li>
+	<li>[O3DE Documentation: Debugging with Lua Editor](https://www.o3de.org/docs/user-guide/scripting/lua/debugging-tutorial/)</li>
+	<li>[O3DE Documentation: Using EBuses in Lua](https://www.o3de.org/docs/user-guide/scripting/lua/ebus/)</li>
+</ul>
```

Comment:
Use MD formatting for lists outside of a table.

Reply:
As an aside.

HTML formatting is only required inside of MD tables. Otherwise MD list formatting works and is preferred.

Reply:
I have changed this and all other similar tables to use MD formatting instead of HTML as requested.